door-runtime/src/jsMain/kotlin/wrappers/SQLiteDatabaseMetadataJs.kt

```diff
@@ -0,0 +1,74 @@
+package wrappers
+
+import com.ustadmobile.door.jdbc.DatabaseMetadata
+import com.ustadmobile.door.jdbc.ResultSet
+import kotlin.text.Regex.Companion.escape
+
+class SQLiteDatabaseMetadataJs(val datasource: SQLiteDatasourceJs): DatabaseMetadata {
+
+    override val databaseProductName: String
+        get() = DB_PRODUCT_NAME_SQLITE
+
+    override fun getTables(
```

Comment:
If this should not be accessible on Javascript, how about taking it out of common?